rudder-core/src/main/scala/com/normation/rudder/domain/policies/RuleTarget.scala

```diff
@@ -387,6 +387,25 @@ object RuleTarget extends Loggable {
     }
   }
 
+
+  /**
+   * Transform a rule target string "id" to
+   * a cfengine class compatible string,
+   * prefixed by "group_"
+   */
+  def toCFEngineClassName(target: String) = {
+    //normalisation process:
+    // 1) to asccii
+    // 2) cfengine normalisation, replacing all non [alphanum-]
+    ///   char by _
+
+    // from http://stackoverflow.com/a/2413228/436331
+    def toAscii(s: String) = java.text.Normalizer.normalize(s, java.text.Normalizer.Form.NFD).replaceAll("""[\p{Mn}\p{Me}]+""", "")
```

Comment:
the reference from stackoverflow doesn't exhibit the same example; is it on purpose ?